hw1/02-palindrome.js

```diff
@@ -1,3 +1,24 @@
-const elem = document.querySelector('input');
+// Sources: https://stackoverflow.com/questions/38053729/javascript-how-to-reverse-a-number
```

Comment:
remove comment

Reply:
Comment is there to indicate a source I referenced for code.